examples/2016-10-31/api_multiple_origin_cors/index.js

```diff
@@ -0,0 +1,67 @@
+// Demonstrates a Lambda function that provides CORS headers for many origins.
+
+const cors = require("./cors-util");
+
+// define a whitelist of allowed origins
+const allowedOrigins = [
+    "http://127.0.0.1",
+    "https://*.example.com",
+    "https://*.amazon.com"
+].map(cors.compileURLWildcards);
```

Comment:
Simpler interface would be `cors.compileURLWildcards([])`. Even better if you just do this for them behind the scenes (inside `cors.makeHeaders`)

Reply:
My goal was to isolate the wildcard support from the rest of the implementation, in case a user wants to use some other library or provide their own regex for the origins.

Reply:
They can easily modify themselves. cors-utils is baked in; not a library.